docs/general/networking/fail2ban.md

```diff
@@ -86,3 +87,28 @@ Assuming you've at least one failed authentication attempt, you can test this ne
 ```bash
 sudo fail2ban-regex /path_to_logs/*.log /etc/fail2ban/filter.d/jellyfin.conf --print-all-matched
 ```
+
+### Optional steps for systemd and nftables
+
+Systemd users like Debian or Ubuntu currently require some additional configs for fail2ban to work.
+Change the file /etc/fail2ban/jail.d/defaults-debian.conf to:
+
+```bash
+[sshd]
+enabled = true
+backend=systemd
+
+[DEFAULT]
+banaction = nftables
+banaction_allports = nftables[type=allports]
+```
+
+Without the telling the sshd jail to use systemd as a backend, fail2ban won't even start. The banaction is to send the bans to nftables instead of unused iptables.
```

Comment:
```suggestion
Without telling the sshd jail to use systemd as a backend, fail2ban won't even start. The `banaction` is set to send the bans to nftables instead of unused iptables.
```

Is this really still necessary on recent Debian/Ubuntu systems?

Reply:
It definitely does still not work out of the box, but I don't know what the best solution for that would be. See last entry here:
https://github.com/fail2ban/fail2ban/discussions/3575